CourseManager/Views/SampleListView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  SampleListView.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-12-01.
+//
+
+import SwiftUI
+
+struct SampleListView: View {
+    var sampleCode = SampleWork(name: "profile", description: "profile view")
+    var cornerRadius = 20.0
```

Comment:
What is this variable for? You use it only in one place 

```
.strokeStyle(cornerRadius: cornerRadius)
```

Reply:
updated